### PR TITLE
Update alerts.mdx

### PR DESCRIPTION
**Pull request by akshayamadhuri**
## Problem

issue: #947 
## Solution

[Provide a detailed explanation of the solution you are proposing, including any relevant code snippets or screenshots.]

## Changes Made

- [List the changes you made in this pull request, including any new features or bug fixes.]

## How to Test

[Provide instructions on how to test the changes you made, including any relevant details like configuration steps or data to be used for testing.]

## Screenshots

[Include screenshots, if relevant, to help reviewers understand the changes you made.]

## Notes

[Any additional notes or information that you would like to share with the reviewers.]

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [ ] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

## Reviewers

@[username of the reviewer]

### CODE REVIEW

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]
Remove these extra lines

File: docs/guides/alerts.mdx [hunk 1]
```diff
@@ -67,3 +67,75 @@ To integrate a custom webhook, you need a URL that listens to the data posted to
...
+
+
+
+
+
+
+
+
+
+
```
Comment (shivam-sharma7) [inline docs/guides/alerts.mdx:141]:
```suggestion






















```

Comment (shivam-sharma7) [reply docs/guides/alerts.mdx:141]: remove these extra lines

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

**Comment by AvineshTripathi** [comment]
Tagging @jakepage91 for review

**Pull request merged**